R/clean_data.R

```diff
@@ -100,7 +101,13 @@ clean_data <- function(data_joined, region_lookup) {
       sleep_onset_time = chron(times = sleep_onset_time),
       sleep_wakeup_time = chron(times = sleep_wakeup_time)
     ) %>%
-    select(-measurementday)
+    select(-measurementday)  %>% 
+    # Fix up the most active time
+    mutate(
+      pa_mostactivehr = 
+        hour(lubridate::ymd_hms(pa_mostactivehr)) + 
```

Comment:
`hour` and `minute` need lubridate attached or, we need lubridate added to requires for this function.

Reply:
Thanks, good catch.

Reply:
Thanks, good catch.